### ISSUE TITLE
[Security] Remove one_time_pads option

### ISSUE THREAD

**Issue by mcdope** [mcdope/pam_usb#178]
I see no usecase for having an option to decrease security. Not that pam_usb would be a security focused project, but we should not actively reduce it...

We should "deprecate" it by, by triggering a `log_error()` if it's active. In the following version we can then remove the option and all related code.

**Comment by steelsnake** [comment]
I can see one reason why: The drive I use has only one partition, and it's fully encrypted. pam_usb does work with this configuration - if OTPs are disabled. I could possibly add a second partition for pam_usb.

**Comment by mcdope** [comment]
Thanks, I wasn't aware of this caveat. 

I assume this is because you need to unlock/open the volume before and are getting errors on pad access/update else?

**Comment by steelsnake** [comment]
Yes, and unlocking the volume can't (normally) happen until logged in.

**Comment by mcdope** [comment]
imho the proper way would be indeed creating a tiny non-encrypted partition for the pads. But I can understand that this can be undesirable. 

tl;dr: option will stay
